src/utils/redux-extension/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './getReduxExtension';
+export * from './createReduxConnection';
```

Comment:
This is just a preference of mine, but I'd avoid `index` stuff.
Can you simply import them directly?